README.md

```diff
@@ -70,9 +70,13 @@ console.log(reactElementToJSXString(<div a="1" b="2">Hello, world!</div>));
 
   Just return the name you want for the provided ReactElement, as a string.
 
-**options.filterProps: array, default []**
+**options.filterProps: string[] | (val: any, key: string) => boolean, default []**
```

Comment:
Is there a good enough reason to deprecate the string array form? It would be trivial for users to switch to the function, but the string form makes what I believe to be the common case pretty simple.